src/components/toast/toast.spec.js

```diff
@@ -189,6 +190,14 @@ describe("Toast", () => {
       elementsTagTest(wrapper.find(IconButton).first().find("span"), ["close"]);
     });
   });
+
+  describe("when toast has specified width", () => {
+    it("should render ToastStyle with correct width and no maxWidth", () => {
+      const wrapper = enzymeFind(mount(<Toast width={200} />), ToastStyle);
+      expect(wrapper).toHaveStyleRule("max-width", undefined);
+      expect(wrapper).toHaveStyleRule("width", "200px");
```

Comment:
I would suggest to use our `assertStyleMatch` function that can be imported from `__spec_helper__/test-utils`
And then the matcher would look like this
```js
assertStyleMatch({
  maxWidth: undefined,
  width: '200px'
})
```